src/d_main.c

```diff
@@ -1710,22 +1710,20 @@ static boolean CheckHaveSSG (void)
 
 static int mainwadfile;
 
+#define SET_DIR(a, b) \
```

Comment:
I think we don't need the `if (a) { free(a); }` construction, I remember adding it a long time ago when I removed the `PATH_MAX` limit. Now I'm pretty sure that we don't allocate these strings anywhere else, so we can simplify this code.

Reply:
The point is that it can be run from the menu multiple times per game now.

Reply:
Okay, got it.